bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitImpl.java

```diff
@@ -183,6 +183,11 @@ private HomekitRoot startBridge(HomekitServer homekitServer, HomekitAuthInfoImpl
         bridges.add(bridge);
         bridge.setConfigurationIndex(changeListener.getConfigurationRevision());
         bridge.refreshAuthInfo();
+        if (settings.useDummyAccessories) {
```

Comment:
just as suggestion, what about to add a condition to the "IF" below instead, e.g.

if (!settings.useDummyAccessorie && currentAccessoryCount < lastAccessoryCount)

it will add 2 more, potentially unnecessary calls for get currentAccessoryCount  and lastAccessoryCount but imo it makes code easier to understand. eitherwise i have 3 times bridge.start in that function.